modules/nw-ovn-k-anp-nbctl.adoc

```diff
@@ -0,0 +1,499 @@
+    - namespaces: # restricted tenants
+        matchLabels:
+          tenant: restricted
+  - name: "default-deny"
+    action: "Deny"
+    to:
+    - networks:
+      - 0.0.0.0/0
+----
+====
+
+.Procedure
+
+. List pods by running the following command:
++
+[source,terminal]
+----
+$ oc get po -n openshift-ovn-kubernetes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                     READY   STATUS    RESTARTS   AGE
+ovnkube-control-plane-5c95487779-8k9fd   2/2     Running   0          33m
+ovnkube-control-plane-5c95487779-v2xn8   2/2     Running   0          33m
+ovnkube-node-524dt                       8/8     Running   0          33m
+ovnkube-node-gbwr9                       8/8     Running   0          24m
+ovnkube-node-h4fpx                       8/8     Running   0          33m
+ovnkube-node-j4hzw                       8/8     Running   0          24m
+ovnkube-node-wdhgv                       8/8     Running   0          33m
+ovnkube-node-wfncn                       8/8     Running   0          24m
+----
+
+. Optional: To list the pods with node information, run the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-ovn-kubernetes -owide
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                     READY   STATUS    RESTARTS   AGE   IP           NODE                                       NOMINATED NODE   READINESS GATES
+ovnkube-control-plane-5c95487779-8k9fd   2/2     Running   0          34m   10.0.0.5     ci-ln-0tv5gg2-72292-6sjw5-master-0         <none>           <none>
+ovnkube-control-plane-5c95487779-v2xn8   2/2     Running   0          34m   10.0.0.3     ci-ln-0tv5gg2-72292-6sjw5-master-1         <none>           <none>
+ovnkube-node-524dt                       8/8     Running   0          33m   10.0.0.4     ci-ln-0tv5gg2-72292-6sjw5-master-2         <none>           <none>
+ovnkube-node-gbwr9                       8/8     Running   0          24m   10.0.128.4   ci-ln-0tv5gg2-72292-6sjw5-worker-c-s9gqt   <none>           <none>
+ovnkube-node-h4fpx                       8/8     Running   0          33m   10.0.0.5     ci-ln-0tv5gg2-72292-6sjw5-master-0         <none>           <none>
+ovnkube-node-j4hzw                       8/8     Running   0          24m   10.0.128.2   ci-ln-0tv5gg2-72292-6sjw5-worker-a-hzbh5   <none>           <none>
+ovnkube-node-wdhgv                       8/8     Running   0          33m   10.0.0.3     ci-ln-0tv5gg2-72292-6sjw5-master-1         <none>           <none>
+ovnkube-node-wfncn                       8/8     Running   0          24m   10.0.128.3   ci-ln-0tv5gg2-72292-6sjw5-worker-b-5bb7f   <none>           <none>
+----
+
+. Navigate into a pod to look at the northbound database by running the following command:
++
+[source,terminal]
+----
+$ oc rsh -c nbdb -n openshift-ovn-kubernetes ovnkube-node-524dt
+----
++
+. Run the following command to look at the `nbdb.ACL`:
++
+[source,terminal]
+----
+$ ovn-nbctl list acl
```

Comment:
Listing ACLs on a big node (or, better said, in a node which is not small) can be a too big output. Most of the times you want to grep (or use `ovn-nbctl find` like in the latest [comments I made on the upstream docs](https://github.com/ovn-org/ovn-kubernetes/pull/4431#discussion_r1644619505)

Reply:
agreed :) list will be noise

Reply:
agreed, I'll add.